### PR TITLE
Goreleaser 

### PR DESCRIPTION
**Pull request by klautcomputing**

### CODE REVIEW

**Comment by klautcomputing** [comment]
Let's tag something tomorrow and push a version :)

```goreleaser --skip-publish --snapshot                                                                                                                                                     
   • running goreleaser 0.32.0
   • loading config file       file=.goreleaser.yml
   • publishing disabled in snapshot mode
   • SETTING DEFAULTS
   • GETTING AND VALIDATING GIT STATE
   • failed to retrieve current tag error=fatal: No names found, cannot describe anything.

   • releasing , commit 12293e50d068fd263ae6b9ef3e1b70f9e9bfa0dd
   • LOADING ENVIRONMENT VARIABLES
   • skipped                   reason=publishing is disabled
   • CHECKING ./DIST
   • BUILDING BINARIES
   • building                  binary=dist/kubeaudit_SNAPSHOT-12293e50d068fd263ae6b9ef3e1b70f9e9bfa0dd_darwin_386/kubeaudit
   • building                  binary=dist/kubeaudit_SNAPSHOT-12293e50d068fd263ae6b9ef3e1b70f9e9bfa0dd_linux_amd64/kubeaudit
   • building                  binary=dist/kubeaudit_SNAPSHOT-12293e50d068fd263ae6b9ef3e1b70f9e9bfa0dd_darwin_amd64/kubeaudit
   • building                  binary=dist/kubeaudit_SNAPSHOT-12293e50d068fd263ae6b9ef3e1b70f9e9bfa0dd_linux_386/kubeaudit
   • CREATING ARCHIVES
   • creating                  archive=dist/kubeaudit_SNAPSHOT-12293e50d068fd263ae6b9ef3e1b70f9e9bfa0dd_linux_amd64.tar.gz
   • creating                  archive=dist/kubeaudit_SNAPSHOT-12293e50d068fd263ae6b9ef3e1b70f9e9bfa0dd_darwin_amd64.tar.gz
   • creating                  archive=dist/kubeaudit_SNAPSHOT-12293e50d068fd263ae6b9ef3e1b70f9e9bfa0dd_linux_386.tar.gz
   • creating                  archive=dist/kubeaudit_SNAPSHOT-12293e50d068fd263ae6b9ef3e1b70f9e9bfa0dd_darwin_386.tar.gz
   • new release artifact      artifact=kubeaudit_SNAPSHOT-12293e50d068fd263ae6b9ef3e1b70f9e9bfa0dd_linux_386.tar.gz
   • new release artifact      artifact=kubeaudit_SNAPSHOT-12293e50d068fd263ae6b9ef3e1b70f9e9bfa0dd_darwin_386.tar.gz
   • new release artifact      artifact=kubeaudit_SNAPSHOT-12293e50d068fd263ae6b9ef3e1b70f9e9bfa0dd_linux_amd64.tar.gz
   • new release artifact      artifact=kubeaudit_SNAPSHOT-12293e50d068fd263ae6b9ef3e1b70f9e9bfa0dd_darwin_amd64.tar.gz
   • CREATING LINUX PACKAGES WITH FPM
   • skipped                   reason=no output formats configured
   • CREATING LINUX PACKAGES WITH SNAPCRAFT
   • skipped                   reason=no summary nor description were provided
   • CALCULATING CHECKSUMS
   • checksumming              file=kubeaudit_SNAPSHOT-12293e50d068fd263ae6b9ef3e1b70f9e9bfa0dd_linux_amd64.tar.gz
   • checksumming              file=kubeaudit_SNAPSHOT-12293e50d068fd263ae6b9ef3e1b70f9e9bfa0dd_linux_386.tar.gz
   • checksumming              file=kubeaudit_SNAPSHOT-12293e50d068fd263ae6b9ef3e1b70f9e9bfa0dd_darwin_amd64.tar.gz
   • checksumming              file=kubeaudit_SNAPSHOT-12293e50d068fd263ae6b9ef3e1b70f9e9bfa0dd_darwin_386.tar.gz
   • new release artifact      artifact=kubeaudit_SNAPSHOT-12293e50d068fd263ae6b9ef3e1b70f9e9bfa0dd_checksums.txt
   • CREATING DOCKER IMAGES
   • building docker image     image=shopify/kubeaudit:SNAPSHOT-12293e50d068fd263ae6b9ef3e1b70f9e9bfa0dd
   • RELEASING TO GITHUB
   • skipped                   reason=--skip-publish is set
   • CREATING HOMEBREW FORMULA
   • skipped                   reason=--skip-publish is set
   • SUCCESS!
```

**Comment by klautcomputing** [comment]
Do we want to add our code of conduct to the tar balls?

**Comment by jinankjain** [comment]
While giving binaries does Code of Conduct matters? Like it is for open source contributions. But may be Jon or Damian would be able to answer that question in a better way.

**Comment by JonPulsifer** [comment]
As a user I wouldn't expect the code of conduct to appear in the binary tars given its purpose is for development, not usage.

It's already included in the "Source code" tar and zip anyway.

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,3 @@
+FROM scratch
+COPY kubeaudit /
```
[the remark this answers is not in the document]
Comment (klautcomputing) [reply Dockerfile]: You mean a multistage build where we first build the binary and then use it in the scratch container?

[the remark this answers is not in the document]
Comment (jinankjain) [reply Dockerfile]: Yeah +1

**Pull request merged**